pkg/vm/vm_test.go

```diff
@@ -359,17 +359,18 @@ func TestCONVERT(t *testing.T) {
 // - array (1) of struct (size+1)
 // which equals to size*2+3 elements in total.
 func appendBigStruct(size uint16) []opcode.Opcode {
-	prog := make([]opcode.Opcode, size*2)
+	prog := make([]opcode.Opcode, size)
 	for i := uint16(0); i < size; i++ {
-		prog[i*2] = opcode.PUSH0
-		prog[i*2+1] = opcode.NEWSTRUCT
+		prog[i] = opcode.NEWSTRUCT0
```

Comment:
This change is certainly not slot-related.

Reply:
Fixed.